processors/load_processor.go

```diff
@@ -72,34 +81,134 @@ func (p loadProcessor) loadMetrics() map[string]loadMetric {
 
 	for _, s := range stats {
 
-		if !strings.Contains(s.Name, ":") {
+		if s.Name == "tcp-load" {
+			metrics["tcp-load"] = loadMetric{
+				metric:   newMetric("load", "tcp_load", "Percentage of TCP children that are awake and processing SIP messages.", []string{}, prometheus.GaugeValue),
+				ip:       "",
+				protocol: "",
+				port:     "",
+				process:  "",
+			}
+			continue
+		} else if strings.Contains(s.Name, "udp:") || strings.Contains(s.Name, "tcp:") {
+			metrics[s.Name] = parseLegacyLoadFormat(s)
+			continue
+		} else if strings.Contains(s.Name, "proc") {
```

Comment:
might be to generic? What are you trying to match?

Reply:
The loads per process come in this format:
```
load:load-proc-3:: 0
load:load1m-proc-3:: 0
load:load10m-proc-3:: 0
```
And in the load module this is the only occurence of `proc`, that's why I'm matching them this way. I'll add some comment to clarify the intent of each `if` statement.